glslang/MachineIndependent/Scan.h

```diff
@@ -130,6 +138,7 @@ class TInputScanner {
                     --chIndex;
                 }
                 loc[currentSource].column = (int)(currentChar - chIndex);
+                logicalSourceLoc.column = (int)(currentChar - chIndex);
```

Comment:
I find it reads better to put them together since they are updated in lockstep: `logicalSourceLoc.column = loc[currentSource].column = ...`. Applies for following cases. But not a big deal.